src/main/groovy/wooga/gradle/paket/unity/internal/UPMPaketPackage.groovy

```diff
@@ -0,0 +1,40 @@
+package wooga.gradle.paket.unity.internal
+
+import groovy.json.JsonOutput
+
+class UPMPaketPackage {
+
+    final File baseDir
+    final String name
+
+    UPMPaketPackage(File paketPackageDir) {
+        this.baseDir = paketPackageDir
+        this.name = baseDir.name
+    }
+
+    Optional<File> getPackageDotJson() {
+        return Optional.ofNullable(new File(baseDir, "package.json")).map {
+            it.exists()? it : null
+        }
+    }
```

Comment:
Please call this the package manifest, which is consistent with the terminology both us and Unity uses.

I would rename this `getPackageManifest`.